js/subsearch_suggestions.js

```diff
@@ -10,12 +10,15 @@
   const urlParams = new URLSearchParams(window.location.search);
   const originalSearch = urlParams.get('original-search');
 
-  let progressText = Drupal.t('Loading suggestions ...');
-  let wrapper = $('#subsearch-suggestions');
-  wrapper.text(progressText);
+  let main = $('.pane-page-content');
+  main.prepend('<div id="subsearch-suggestions" style="background-color: #F1F2F2; padding-top: 30px;"><div style="max-width: 1124px;margin-left: auto;margin-right: auto;width: 90%;" class="suggestions inner-content"></div></div>');
```

Comment:
Ideally, this approach should be avoided. Please a TODO section explaining why we need this way and as a notice to move this to a certain template/preprocess.
I notice it just appends this element whenever this javascript is attached.